kola/tests/update/update.go

```diff
@@ -52,6 +53,15 @@ func init() {
 			return kola.UpdatePayloadFile == ""
 		},
 	})
+	register.Register(&register.Test{
+		Name:        "cl.sysext.boot",
+		Run:         sysextBootLogic,
+		ClusterSize: 0,
+		Distros:     []string{"cl"},
+		// This test is uses its own OEM files and shouldn't run on other platforms
+		Platforms:  []string{"qemu", "qemu-unpriv"},
+		MinVersion: semver.Version{Major: 9999},
```

Comment:
Ha ha ha. Alright, I guess it will be fine to merge this now and update the min version when changes in bootengine are merged.